fc/jira/triage_task.py

```diff
@@ -113,10 +114,12 @@ def create(self):
     def type_str(self) -> str:
         return 'Triage'
 
-    def score(self) -> int:
+    def score(self) -> tuple:
```

Comment:
You can use another, more specific version of typing here.
Put `from typing import Tuple` in the import statements at the top.
And then here...
```suggestion
    def score(self) -> Tuple[int, bool]:
```

Reply:
Also, this is an inherited function from `FcIssue`.  You'll need to update that signature and any other inheriting classes (seems to be `TriageTask` and `BacklogStory`).